src/lib/services/email-service.ts

```diff
@@ -101,7 +101,7 @@ export class EmailService {
                 text: bodyText,
             };
             process.nextTick(() => {
-                this.mailer.sendMail(email).then(
+                this.mailer!.sendMail(email).then(
```

Comment:
can we solve it without "!"  ?

Reply:
I think not, because Typescript does not know we're validating that mailer is not undefined inside `if (this.configured()) {`

Reply:
ah because it's a class field and can be set to undefined any time

Reply:
Do you have any idea how to solve this? I think we can't make it a constant :thinking: can't we?

Reply:
I tried a few things but I think this is ok for now

Reply:
Likewise :P